### ISSUE TITLE
PIE-related GCC compile errors when using more recent compilers

### ISSUE THREAD

**Issue by mrayva** [dbtoaster/dbtoaster-backend#95]
gcc 5.4 on Ubuntu 16.04 was compiling ok a couple of month back, but gcc 7.3 on Ubuntu 18.04 gives me problems.  I feel that it is "position-independent executables"-related but can't pinpoint exact place in your scripts. 

[error] /usr/local/opt/dbtoaster-a5//bin/dbtoaster_release: Symbol `acos' causes overflow in R_X86_64_PC32 relocation
[error] /usr/local/opt/dbtoaster-a5//bin/dbtoaster_release: Symbol `atan2' causes overflow in R_X86_64_PC32 relocation
[error] /usr/local/opt/dbtoaster-a5//bin/dbtoaster_release: Symbol `cos' causes overflow in R_X86_64_PC32 relocation
[error] /usr/local/opt/dbtoaster-a5//bin/dbtoaster_release: Symbol `cos' causes overflow in R_X86_64_PC32 relocation
[error] /usr/local/opt/dbtoaster-a5//bin/dbtoaster_release: Symbol `pow' causes overflow in R_X86_64_PC32 relocation
[error] /usr/local/opt/dbtoaster-a5//bin/dbtoaster_release: Symbol `pow' causes overflow in R_X86_64_PC32 relocation
[error] /usr/local/opt/dbtoaster-a5//bin/dbtoaster_release: Symbol `pow' causes overflow in R_X86_64_PC32 relocation
[error] /usr/local/opt/dbtoaster-a5//bin/dbtoaster_release: Symbol `pow' causes overflow in R_X86_64_PC32 relocation
[error] /usr/local/opt/dbtoaster-a5//bin/dbtoaster_release: Symbol `atan' causes overflow in R_X86_64_PC32 relocation

**Comment by sachinbjohn** [comment]
I encountered the issue a few days ago. This can be fixed by adding "-ccopt -no-pie" to OPT_FLAGS in makefile.inc in the dbtoaster-a5 repo

**Comment by mrayva** [comment]
OPT_FLAGS  in makefile.inc is for OCAML compiler use, not gcc...?

**Comment by sachinbjohn** [comment]
You are right. But technically, ocamlopt uses gcc to link the object files and -ccopt forwards the arguments to gcc. I just happened to have the same error, and adding this fixed it. If that didn't work for you, I don't have any more suggestions. Do you @losmi83  ?
